src/backend/distributed/shared_library_init.c

```diff
@@ -509,6 +521,17 @@ RegisterCitusConfigVariables(void)
 		0,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
+		"citus.log_local_commands",
+		gettext_noop("Log queries that are executed locally, can be overriden by "
+					 "citus.log_remote_commands"),
```

Comment:
Maybe just rely on the 1 GUC? The extra GUC just seems like a pain.

Reply:
I agree, two notes:

(a) `citus.log_remote_commands` overrules `citus.log_local_commands` when it is true. So, when you have `citus.log_remote_commands` set to ON, you'll see both local and remote commands.

(b) The tricky part is that there is no easy way of showing that the execution is done locally. And, if I set `citus.log_remote_commands`, then the regression output becomes extremely verbose. That's why I came up with this trick, which is probably only useful for testing.

If you still think this is not good, I can find another way for understanding the local query executions on the regression tests.

Reply:
(And, it is `GUC_NO_SHOW_ALL`)

Reply:
fair